test/behaviour/concept/type/thingtype/ThingTypeSteps.java

```diff
@@ -318,12 +318,10 @@ public void thing_type_set_plays_role_throws_exception(RootLabel rootLabel, Stri
         assertThrows(() -> get_thing_type(rootLabel, typeLabel).setPlays(roleType));
     }
 
-    @When("{root_label}\\( ?{type_label} ?) set plays role: {scoped_label} as {type_label}")
-    public void thing_type_set_plays_role_as(RootLabel rootLabel, String typeLabel, Parameters.ScopedLabel roleLabel, String overriddenLabel) {
+    @When("{root_label}\\( ?{type_label} ?) set plays role: {scoped_label} as {scoped_label}")
+    public void thing_type_set_plays_role_as(RootLabel rootLabel, String typeLabel, Parameters.ScopedLabel roleLabel, Parameters.ScopedLabel overriddenLabel) {
```

Comment:
Previously we allowed unscoped role labels here, but only in this step- not in any others. This was inconsistent, so we now require scoped labels, which also simplifies the implementation.